Samples/PublicSamples/RecordingBot/src/RecordingBot.Services/ServiceSetup/ServiceHost.cs

```diff
@@ -16,8 +17,13 @@ public class ServiceHost : IServiceHost
 
         public ServiceHost Configure(IServiceCollection services, IConfiguration configuration)
         {
-            Services = services;
-            Services.AddSingleton<IGraphLogger, GraphLogger>(_ => new GraphLogger("RecordingBot", redirectToTrace: true));
+            Services = services;      
+            Services.AddSingleton<IGraphLogger, GraphLogger>(_ =>
```

Comment:
Use proper name for service provider variable, because you are actually using it and not just throwing it away.

Reply:
Happens on other calls as well...
maybe we want to have a PR and US for overall Code Convention "fixes"